models/connexus_user.py

```diff
@@ -1,5 +1,24 @@
+import logging
+
+from stream import Stream
 from google.appengine.ext import ndb
 
 class ConnexusUser(ndb.Model):
     user_id = ndb.StringProperty()
     report = ndb.IntegerProperty()
+    streams_subscribed = ndb.KeyProperty(Stream, repeated=True)
+
+    def get_subscribed_streams(self):
+        streams = ndb.get_multi(self.streams_subscribed)
+        return streams
+
+    @classmethod
+    def from_user_id(cls, user_id):
+        return cls.query().filter(cls.user_id == str(user_id)).get()
+
+    @classmethod
+    def is_subscribed(cls, user_id, stream_key):
```

Comment:
This reminds me I need a get method for the report to show properly in the Trending screen.